cmd/posix-list-dir_unix.go

```diff
@@ -90,9 +90,9 @@ func parseDirents(dirPath string, buf []byte) (entries []string, err error) {
 				return nil, err
 			}
 			if fi.IsDir() {
-				entries = append(entries, fi.Name()+slashSeparator)
+				entries = append(entries, name+slashSeparator)
```

Comment:
@harshavardhana I guess we need also to apply this change to plan9

Reply:
No plan9 is already doing it correctly we use fi.Name() the buggy was Linux and Windows code alone.